metadata.json

```diff
@@ -78,11 +78,11 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.10.0 < 9.0.0"
+      "version_requirement": ">= 7.0.0 < 9.0.0"
```

Comment:
This drops a significant amount of major versions. Maybe that can be done in a dedicated PR so it's more present in the changelog?

Reply:
I believe that was in the squashed #344
Is this a blocker? I can't see how it would matter. No reasonable way to test on agents that old.

Reply:
if it matters depends on the CHANGELOG.md quality Puppet wants to have. For Vox Pupuli it's important to document those breaking changes. I suggest to make this change in a dedicated PR and rebase your `good-times` branch against it.